src/clib/pio_nc4.c

```diff
@@ -1112,6 +926,11 @@ int PIOc_set_var_chunk_cache(int ncid, int varid, size_t size, size_t nelems,
     ios = file->iosystem;
     msg = PIO_MSG_SET_VAR_CHUNK_CACHE;
 
+    int my_rank;
+    int ret;
+    if ((ret = MPI_Comm_rank(MPI_COMM_WORLD, &my_rank)))
```

Comment:
Those were for debugging purposes and should have been removed. They are removed now.